comit/src/expiries.rs

```diff
@@ -0,0 +1,631 @@
+//! Standardized COMIT Expiry Times
+//!
+//! The COMIT protocols rely on two expiry times in order for the swap to be
+//! atomic. One expiry for the alpha ledger and one for the beta ledger. The
+//! beta ledger expiry time must elapse before the alpha ledger expiry time.
+
+use crate::timestamp::{self, Timestamp};
+use time::{prelude::*, Duration};
+
+// TODO: Currently we ignore deploy for Erc20.
+
+/// Current time as a UNIX timestamp from the perspective of the implementer.
+///
+/// Intended for getting the current time from the underlying blockchain.
```

Comment:
```suggestion
/// Intended for getting the current time from the underlying blockchain.
/// The definition `current time` varies depending on the blockchain, this always
/// refers to the time used by the op codes used in the COMIT contracts.
```
Maybe something to clarify here.

Reply:
I do not fully understand the last line but I used your text as is.

Reply:
Well, don't use it if you don't understand it... What I am saying is that the definition of `current time` differs from blockchain to blockchain and that what we define as `current time` here is what is considered as `current time` by the op codes of EVM or Bitcoin Script (meaning latest block time and median time).

Maybe it's an unnecessary clarification that just confuses more than it helps.